src/scripts/components/Chart.jsx

```diff
@@ -66,30 +55,33 @@ class Chart extends React.Component {
   );
 
   render() {
-    const { ...ChartProperties } = this.props;
+    let hasData = true;
+    if (typeof this.props.data === 'undefined' || this.props.data === null || Object.keys(this.props.data).length === 0 || this.props.data.datasets.length === 0) {
+      hasData = false;
+    }
```

Comment:
Consider using const for better readability:

`const hasData = this.props.data && Object.keys(this.props.data).length && this.props.data.datasets && this.props.data.datasets.length`

Reply:
still gross looking but not much you can do there...

Reply:
👍 done. thanks